samples/sampler/stories/qa/Marquee.js

```diff
@@ -1,310 +1,359 @@
+	render: ({updateLocale}) => (
+		<div>
+			<Heading showLine>Remeasure marquee when locale change causes a font change with different metrics</Heading>
+			<Button onClick={updateLocale}>Change locale and marquee stops</Button>
+		</div>
+	)
+}));
+
+const CustomItemBase = ({children, ...rest}) => (
+	<div {...rest} style={{display: 'flex', width: 300, alignItems: 'center'}}>
+		<Icon>flag</Icon>
+		<Marquee id="marqueeText" style={{flex: 1, overflow: 'hidden'}}>{children}</Marquee>
+		<Icon>trash</Icon>
+	</div>
+);
+
+const CustomItem = Spottable(MarqueeController(
+	{marqueeOnFocus: true},
+	CustomItemBase
+));
+
+const MarqueeItem = Spottable(
+	MarqueeController(
+		{marqueeOnFocus: true},
+		ItemBase
+	)
+);
+
+const MarqueeWithShortContent = () => {
+	const [long, setLong] = useState(false);
+	const [node, setNode] = useState(null);
+	const [scrollWidth, setScrollWidth] = useState(null);
+	const [width, setWidth] = useState(null);
+
+	useEffect(() => {
+		setNode(document.querySelector('#marqueeText'));
+		if (!node) {
+			return;
+		}
+		if (node.scrollWidth !== scrollWidth) {
+			setScrollWidth(node.scrollWidth);
+			setWidth(node.getBoundingClientRect().width);
+		}
+	}, [node, scrollWidth]);
+
+	const handleClick = useCallback(() => setLong(!long), [long]);
+
+	return (
+		<div>
+			scrollWidth: {scrollWidth} width: {width}
+			<CustomItem onClick={handleClick}>{long ? 'Very very very very very very very very very long text' : 'text'}</CustomItem>
+		</div>
+	);
+};
+
+const MarqueeWithContentChanged = () => {
+	const [count, setCount] = useState(0);
+
+	const handleClick = useCallback(() => {
+		setCount((count + 1) % 3);
+	}, [count]);
+
+	return (
+		<div>
+			<ol>
+				<li>Click once to show the ellipsis just before the text marquees the first time.</li>
+				<li>Click a second time to show the ellipsis just before the text marquees the first time</li>
+				<li>Click again to return to a short string without marquee.</li>
+			</ol>
+			<Button onClick={handleClick}>
+				{'Click Me'}
+			</Button>
+			<Marquee marqueeOn={'render'} style={{width: '400px'}}>{texts[count]}</Marquee>
+		</div>
+	);
+};
+
+export default {
+	title: 'Moonstone/Marquee',
+	component: 'Marquee'
+};
+
+export const WithLTR = (args) => {
+	const disabled = args['disabled'];
+	return (
+		<section>
+			<Marquee
+				disabled={disabled}
+				forceDirection={args['forceDirection']}
+				marqueeDelay={args['marqueeDelay']}
+				marqueeDisabled={args['marqueeDisabled']}
+				marqueeOn={args['marqueeOn']}
+				marqueeOnRenderDelay={args['marqueeOnRenderDelay']}
+				marqueeResetDelay={args['marqueeResetDelay']}
+				marqueeSpeed={args['marqueeSpeed']}
+				style={{width: ri.unit(399, 'rem')}}
+			>
+				{args['children']}
+			</Marquee>
+			{disabledDisclaimer(disabled)}
+		</section>
+	);
+};
+
+select('children', WithLTR, LTR, Marquee, LTR[0]);
+boolean('disabled', WithLTR, Marquee, false);
+select('marqueeOn', WithLTR, ['hover', 'render'], Marquee, 'render');
```

Comment:
The 'marqueeOn' select is duplicated and 'forceDirection' is missing. 
```suggestion
select('forceDirection', WithLTR, ['', 'ltr', 'rtl'], Marquee, '');
```